docs/index.asciidoc

```diff
@@ -38,6 +38,45 @@ type of event can be enriched and transformed with a broad array of input, filte
 native codecs further simplifying the ingestion process. Logstash accelerates your insights by harnessing a greater
 volume and variety of data.
 
+[float]
+[[right-option]]
+=== Is Logstash the right ingest option?
```

Comment:
I like that you're addressing this question in the docs because our current docs make it hard for users to figure out (at a glance) the best way to ingest log data. As Tanya has pointed out, our product naming causes people to think Logstash for logs. Also the organization of our content on the doc landing page (organized by product rather than user goals) makes it challenging for users to know where to go first.

I think you're ideas, but some of the details are a bit misleading. For example:

- Filebeat modules offer more than your description suggests. They don't just make configuration easier, they also come with sample dashboards, an index template, and, in some cases, machine learning jobs.
- The distinction you make between Filebeat modules and ingest node might confuse users because Filebeat modules actually *use* ingest node to parse the data collected by Filebeat.
- Some users will want to use Filebeat modules to collect and parse the data but still send events to Logstash (they can configure Logstash to read from the ingest pipelines). They might do this if they want to enrich the data before indexing it into Elasticsearch. Another use case is when users want to route messages through Kafka (Filebeat > Kafka > Logstash > Elasticsearch). They can still use the ingest pipelines provided by Filebeat to parse the data, but they'll need Logstash to read the messages from Kafka.

There are also users who might want to use Logstash instead of Filebeat because Logstash offers inputs, filters, and outputs that aren't provided by Filebeat (see the [FAQ topic](https://www.elastic.co/guide/en/beats/filebeat/current/faq.html#diff-logstash-beats) I wrote. It's a bit simplistic, but is a start).

I think some input from the PM team (and maybe support) would be very useful to make sure the details we provide here are helpful and accurate. 

I also wonder if the LS docs are the right place for this. What I'd like to see is an overview (maybe in the stack docs) that describes the ingest options and walk users through the decision making process. Maybe even providing some kind of decision tree would make sense.

Reply:
This is great feedback. I'm off to incorporate your ideas and comments.  
Thanks for taking the time to review.